Restoring and improving the UT for log-level endpoint

asifdxtreme opened the pull request:
Restored the unit test cases for log-level endpoint.
Improved the assertion of the log messages for each acceptable forms.
This PR tries to resolve https://www.pivotaltracker.com/story/show/91912146

shashidharatd commented on cf_debug_server_test.go:
```diff
@@ -113,5 +115,46 @@ var _ = Describe("CF Debug Server", func() {
+						Ω(err).ShouldNot(HaveOccurred())
+
+						sink.Log(level, []byte("hello before level change"))
+						Ω(logBuf).ShouldNot(gbytes.Say("hello before level change"))
```
How will this assert pass? when we are iterating with all log levels in for loop. some log level should output the log?
Any explanation?

asifdxtreme replied:
At the time of initialization of debug server there is default log level assigned to it. https://github.com/cloudfoundry-incubator/cf-debug-server/blob/master/server.go#L40

shashidharatd commented on cf_debug_server_test.go:
```diff
@@ -113,5 +115,46 @@ var _ = Describe("CF Debug Server", func() {
+						response, err := http.DefaultClient.Do(request)
+						Ω(err).ShouldNot(HaveOccurred())
+
+						Ω(response.StatusCode).Should(Equal(http.StatusOK))
```
Use Expect instead of Omega. I have seen in garden-linux one commit where they do this. they would prefer Expect matcher then omega

asifdxtreme commented:
@shashidharatd  @dhilipkumars the changes has been done as per review comments...please check

dhilipkumars commented:
@asifhuawei  Looks good to me.

asifdxtreme committed:
Restoring and improving the UT for log-level endpoint

Restored the unit test cases for log-level endpoint.
Improved the assertion of the log messages for each acceptable forms.
This PR tries to resolve https://www.pivotaltracker.com/story/show/91912146

shashidharatd commented:
Merged. plz raise the same to cf.
